src/main/java/com/kuljava/swiatwsi/rawmaterials/RawMaterials.java

```diff
@@ -0,0 +1,34 @@
+package com.kuljava.swiatwsi.rawmaterials;
+
+import com.kuljava.swiatwsi.world.Village;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "raw_materials")
+@Getter
+@Setter
+@ToString
+public class RawMaterials implements Serializable {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+    @Column(name = "wood_quantity")
+    private Long woodQuantity;
+    @Column(name = "clay_quantity")
+    private Long clayQuantity;
+    @Column(name = "iron_quantity")
+    private Long ironQuantity;
+
+    public RawMaterials() {
+        this.woodQuantity = 0l;
+        this.clayQuantity = 0l;
+        this.ironQuantity = 0l;
```

Comment:
This is business logic. Placing any business logic in the constructor (especially non-parametrized) is very bad practice. If You want to do such a thing, think about static factory method, or factory class. Non parametrized constructor should only create empty object (if it is possible). Parametrized constructors should ONLY assign parameters to values.

Reply:
Ok, I will change it